src/Workouts.svelte

```diff
@@ -14,30 +14,48 @@
         let baseWeight = Math.min(...c.weights);
 
         for (let weight of c.weights) {
-            let reps = c.proportionateReps ? Math.floor(baseWeight / weight * c.baseReps) : c.baseReps;
-
+            let base_reps = c.proportionateReps ? Math.floor(baseWeight / weight * c.baseReps) : c.baseReps;
+            let max_volume = 0;
             for (let sets = c.minSets; sets <= c.maxSets; sets += c.delta) {
+                // volume cycle: Reps = base reps
+                const reps = base_reps;
+                const volume = weight * sets * reps;
                 workouts.push({
                     weight,
                     sets,
                     reps,
-                    volume: weight * sets * reps
+                    volume,
+                    is_volume_cycle: true
                 });
+                max_volume = Math.max(max_volume, volume);
```

Comment:
this makes sense - to keep the volume constant once you enter the density cycle. but you'll want to keep track of volume for each weight separately. so only need to calculate c.maxReps * base_reps